understanding-asynchronous-programming/example_3.py

```diff
@@ -1,15 +1,17 @@
 import time
 import queue
-from lib.elapsed_time import ET
+from codetiming import Timer
 
 
 def task(name, queue):
+    text = ''.join([f'Task {name} elapsed time: ', '{:.2f}'])
+    timer = Timer(text=text)
```

Comment:
Ah, I see you ran into the same f-string-and-`.format` problem that had me stumped for a while :blush: 

You can escape curly braces in f-strings by doubling them, so `f"{{:.2f}}"` evaluates as `"{:.2f}"`. The instantiation of the timer can then be done with:

```
timer = Timer(text=f"Task {name} elapsed time: {{:.2f}}")
```

Reply:
I also note that in your current `ET` code you print out with one decimal, while here you've changed to 2. At least on my computer, I then get results like `Task One elapsed time: 15.01` which doesn't look quite as clean and might distract a little from the focus of the article?

I would therefore suggest going back to one decimal by using `.1f` instead of `.2f`.

Reply:
Finally, I noticed the linter was failing on your updates. This seemed to be solely due to `black` wanting to have double quotes instead of single quotes. I've used double quotes in my suggested code above.

Reply:
Changed all three to your recommendations, though I'm having trouble getting black to run. Keeps giving me a 

```
dyld: Library not loaded: @executable_path/../.Python
  Referenced from: /usr/local/Cellar/black/19.3b0/libexec/bin/python3.7
  Reason: image not found
```
error. But I changed to double-quotes just with a search and replace as there were only a few.